manifest/process.go

```diff
@@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
 		}
 	}
 
+	for i, p := range s.Ports {
+		// Assign random ports to "internal" balancers
+		if !p.Public {
+			p.Public = true
```

Comment:
I agree, process/run/sync should be in a run package or even the cli package.
As always the refactor here is a step change.

I have just been using the rails example and done exactly that (double postgres).
The original issue is here: https://github.com/convox/rack/issues/573 it has been closed but I believe the problem persists for certain use cases?? cc @mattmanning